unit/goto-programs/goto_program_assume.cpp

```diff
@@ -38,7 +38,7 @@ SCENARIO(
     symbol_table.insert(symbol);
     symbol_table.insert(fun_symbol);
     namespacet ns(symbol_table);
-    instructions.back().make_assertion(x_le_10);
+    instructions.back() = goto_programt::make_assertion(x_le_10);
```

Comment:
Can we actually remove the `emplace_back` above and use `add` instead?

Reply:
sure